content/assets/stylesheets/includes/mixins.scss

```diff
@@ -0,0 +1,16 @@
+@mixin text-rotate($rotation) {
```

Comment:
We have an autoprefixer in place which adds all of these prefixes, so in theory you could simply use the transform + rotation directly in the CSS and it should work fine for all browsers.

Reply:
Ah cool, what do we use for that?

Reply:
See https://github.com/ZeusWPI/zeus.ugent.be/blob/master/lib/filters/autoprefixer.rb and  https://github.com/ZeusWPI/zeus.ugent.be/blob/master/Rules#L181

It's Rails' prefixer